rust/rsc/src/rsc/add_job.rs

```diff
@@ -12,13 +12,15 @@ use tracing;
 // The actual max is 65536, but adding an arbritrary buffer of 36 for any incidental parameters
 const MAX_SQLX_PARAMS: usize = 65500;
 
-#[tracing::instrument]
+#[tracing::instrument(skip_all)]
 pub async fn add_job(
     Json(payload): Json<AddJobPayload>,
     conn: Arc<DatabaseConnection>,
 ) -> StatusCode {
     // First construct all the job details as an ActiveModel for insert
     let hash = payload.hash();
+    tracing::info!(hash);
```

Comment:
Does this include the the function name or something?  It seems like it must otherwise just printing the hash seems hard to use.

Reply:
Its just the hash which is mainly useful for seeing differences between repeated log lines, but also its a key into the database so we can look up the hash in the db if needed